Codeinwp/neve
# chore: moving pro starter sites definition into the free version

[Codeinwp/neve-pro-addon#682]
[Codeinwp/neve-pro-addon#695]
[Codeinwp/neve#1603]

### Summary
Moved all starter sites definition into the theme.
Added notice when rest API is not available when mounting sites library about page tab.
Parse changelog markdown formatted links properly.

### Will affect visual aspect of the product

NO

### Test instructions


- Only works properly with this PR: https://github.com/Codeinwp/neve-pro-addon/pull/703
- On old pro, the sites will still show as upsells.
- With the rest API unavailable, the about page should display a notice instead of the sites list. ([Codeinwp/neve-pro-addon#695])
- I've corrected a few past releases here and in the aforementioned PR and they should be displayed properly like in the screenshot [here](https://github.com/Codeinwp/neve/issues/1603#issuecomment-630859896)


Closes Codeinwp/neve-pro-addon#682, Codeinwp/neve-pro-addon#695, #1603.
